### DISCUSSION TITLE
Truly Working on M1 MacBook Pro with Big Sur?

### DISCUSSION THREAD

**Discussion by ChrisPirillo** [Caldis/Mos#359] [Q&A]
I'm not quite sure the application is currently working as intended on the latest build of MacOS 11.2.1 (20D74) - or on any Big Sur installation, specifically on an M1 system. Adjusting sliders appears to have absolutely no impact on either Touchpad or Mouse acceleration (and Scroll Monitor doesn't appear to illustrate any differences with configurations, either). 

Obviously, this is an Intel app, but I don't know if that's where the disconnect is - or if it's another compatibility issue at play? Am I missing something?

**Comment by Caldis** [comment]
As I mentioned in change log, currently I not upgrade my devices to M1 versions so I not sure what's going on in the M1 device ...

**Comment by ChrisPirillo** [comment]
Is there anything I might be able to do to help on my end?
________________________________

**Comment by pan93412** [comment]
Mos is already a universal application that works natively on Apple Silicon, and I can confirm it works properly :)

**Reply by pawelklimiuk** [reply]
could you try to instal it on big sur 11.5.1?

**Comment by lemesevkirill** [comment]
M1 OS:12.0.1
I can't start MOS. OS message: Mos.app can't be opened because Apple cannot check it for malicious software. This software needs to be updated.
